tools/pytorch2onnx.py

```diff
@@ -52,10 +53,31 @@ def pytorch2onnx(config_path,
         opset_version=opset_version)
 
     model.forward = orig_model.forward
+
+    # simplify onnx model
+    if do_simplify:
+        from packaging import version
+        import mmcv
+        min_required_version = '1.2.5'
+        assert version.parse(mmcv.__version__) >= version.parse(
+            min_required_version
+        ), f'Requires to install mmcv>={min_required_version}'
+        from mmcv.onnx.simplify import simplify
+
+        input_dic = {'input': one_img.detach().cpu().numpy()}
+        _ = simplify(output_file, [input_dic], output_file)
     print(f'Successfully exported ONNX model: {output_file}')
     if verify:
-        from mmdet.core import get_classes
+        from mmdet.core import get_classes, bbox2result
         from mmdet.apis import show_result_pyplot
+
+        ort_custom_op_path = ''
+        try:
+            from mmcv.ops import get_onnxruntime_op_path
+            ort_custom_op_path = get_onnxruntime_op_path()
+        except (ImportError, ModuleNotFoundError):
+            pass
```

Comment:
If we pass here, what would happen? Should we raise an warning?

Reply:
Added a warning. It's Ok to pass if the input model does not have a custom op from `mmcv`.